src/main/java/practice/StreamPractice.java

```diff
@@ -1,29 +1,48 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
+    private static final String SPLIT_SYMBOL = ",";
     /**
      * Given list of strings where each element contains 1+ numbers:
      * input = {"5,30,100", "0,22,7", ...}
      * return min integer value. One more thing - we're interested in even numbers.
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(e -> e.split(SPLIT_SYMBOL))
+                .flatMap(Arrays::stream)
+                .map(Integer::parseInt)
+                .filter(i -> i % 2 == 0)
```

Comment:
For the future, do not use the abbreviation of the name
```suggestion
                .map(line -> line.split(SPLIT_SYMBOL))
                .flatMap(Arrays::stream)
                .map(Integer::parseInt)
                .filter(number -> number % 2 == 0)
```